config.js

```diff
@@ -702,6 +702,12 @@ config.TIERING_TTL_WORKER_BATCH_SIZE = 1000;
 config.TIERING_TTL_WORKER_BATCH_DELAY = 1 * 60 * 1000; // 1 minutes
 config.TIERING_TTL_MS = 30 * 60 * 1000; // 30 minutes
 
+//////////////////////////
+//         TMFS         //
+//////////////////////////
+
+config.TMFS_BUCKET_TIER2_AUTOCONF = false;
```

Comment:
I think this config should actually be defined as a generic bucket tiering option, and not specific to TMFS.
This will allow us to use it also for other buckets such as with S3 backing stores that can automatically tier to secondary storage class.
Meaning move it to be somewhere in the // SYSTEM CONFIG // area, like:
```suggestion
config.BUCKET_AUTOCONF_TIER2_ENABLED = false;
```